src/scripts/dashboardpage.js

```diff
@@ -1,447 +1,928 @@
+        });
     }
 
-    function onActiveDevicesClick(e) {
-        var btn = dom.parentWithClass(e.target, "sessionCardButton");
+    function onActiveDevicesClick(e__r) {
+        var btn = dom.parentWithClass(e__r.target, "sessionCardButton");
+
         if (btn) {
             var card = dom.parentWithClass(btn, "card");
+
             if (card) {
-                var sessionId = card.id,
-                    session = (DashboardPage.sessionsList || []).filter(function(s) {
-                        return "session" + s.Id === sessionId
-                    })[0];
-                session && (btn.classList.contains("btnCardOptions") ? showOptionsMenu(btn, session) : btn.classList.contains("btnSessionInfo") ? showPlaybackInfo(btn, session) : btn.classList.contains("btnSessionSendMessage") ? showSendMessageForm(btn, session) : btn.classList.contains("btnSessionStop") ? connectionManager.getApiClient(session.ServerId).sendPlayStateCommand(session.Id, "Stop") : btn.classList.contains("btnSessionPlayPause") && session.PlayState && connectionManager.getApiClient(session.ServerId).sendPlayStateCommand(session.Id, "PlayPause"))
+                var sessionId = card.id;
+                var session = (DashboardPage.sessionsList || []).filter(function (s__t) {
+                    return "session" + s__t.Id === sessionId;
+                })[0];
+
+                if (session) {
+                    if (btn.classList.contains("btnCardOptions")) {
+                        showOptionsMenu(btn, session);
+                    } else {
+                        if (btn.classList.contains("btnSessionInfo")) {
+                            showPlaybackInfo(btn, session);
+                        } else {
+                            if (btn.classList.contains("btnSessionSendMessage")) {
+                                showSendMessageForm(btn, session);
+                            } else {
+                                if (btn.classList.contains("btnSessionStop")) {
+                                    connectionManager.getApiClient(session.ServerId).sendPlayStateCommand(session.Id, "Stop");
+                                } else {
+                                    if (btn.classList.contains("btnSessionPlayPause") && session.PlayState) {
+                                        connectionManager.getApiClient(session.ServerId).sendPlayStateCommand(session.Id, "PlayPause");
+                                    }
+                                }
+                            }
+                        }
+                    }
+                }
             }
         }
     }
 
     function filterSessions(sessions) {
-        for (var list = [], minActiveDate = (new Date).getTime() - 9e5, i = 0, length = sessions.length; i < length; i++) {
-            var session = sessions[i];
+        var list = [];
+        var minActiveDate = new Date().getTime() - 9e5;
+
+        for (var i__y = 0, length = sessions.length; i__y < length; i__y++) {
```

Comment:
why change `i` to `i__y`?..

Reply:
I'm guessing the deuglify lib tries to make sure there are no clashes. I've been manually fixing some of them.